addons/project/views/project_views.xml

```diff
@@ -1669,6 +1669,18 @@
             </field>
         </record>
 
+        <record id="open_view_my_tasks_list_view" model="ir.ui.view">
+            <field name="name">Name</field>
+            <field name="model">project.task</field>
+            <field name="inherit_id" ref="view_task_tree2"/>
+            <field name="mode">primary</field>
+            <field name="arch" type="xml">
+                <field name="stage_id" position="replace">
+                    <field name="stage_display" optional="show" readonly="1"/>
```

Comment:
```suggestion
                    <field name="stage_display" optional="show"/>
```
The readonly is actually not necessary, since the field is computed, and therefore already readonly (my bad)

Reply:
Done :)